Resources/Prototypes/Entities/Objects/Materials/materials.yml

```diff
@@ -182,3 +182,27 @@
     state: cotton
   - type: Stack
     count: 1
+
+- type: entity
+  parent: MaterialBase
+  id: FissionFuel
+  name: fission fuel
+  suffix: Full
+  description: Radioactive Fuel made for Nuclear Fission Engines.
```

Comment:
Since this doesn't actually seem to be used for anything, it's out-of-scope for this PR. Keep the reagents used by the new explosion and remove anything unused

Reply:
Yeah